doc/reference/configuration/cfg_storage.rst

```diff
@@ -70,11 +92,34 @@
     memory depending on the total amount of memory available and the
     distribution of item sizes. Allowed values range from 1 to 2.
 
+    See also: :ref:`slab_alloc_granularity <cfg_storage-slab_alloc_granularity>`
+
     | Type: float
     | Default: 1.1
     | Environment variable: TT_SLAB_ALLOC_FACTOR
     | Dynamic: no
 
+.. _cfg_storage-slab_alloc_granularity:
+
+.. confval:: slab_alloc_granularity
+
+    Since version :doc:`2.8.1 </release/2.8.1>`.
+    Specifies the granularity of memory allocation in the :ref:`small allocator <cfg_storage-memtx_allocator>`.
+    The value of ``slab_alloc_granularity`` should be a power of two and should greater than or equal to 4.
```

Comment:
```suggestion
    The value of ``slab_alloc_granularity`` should be a power of two and should be greater than or equal to 4.
```

Reply:
BTW, is this value an actual memory size in some units or an abstract parameter? 
In the first case, we should mention the unit in this description.

Reply:
Good point, @alyapunov could you please clarify regarding this question?

Reply:
It is measured in bytes.

Reply:
thanks!